src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo-List</title>
+    <link rel="stylesheet" href="../src/index.css" />
+    <script src="https://kit.fontawesome.com/043c3d901b.js" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+
+  </head>
+  <body>
+    <form class="todo">
+      <div class="todo-list-title">
+        <p> Todo List</p>
+        <i id="icon" class="fas fa-sync recycle"></i>
+      </div>
+      <input type="text" class="activity" placeholder="Add to your list..." />
+      <ul class="todo-list"></ul>
+      <button type="button" class="clear bg-color border-0">Clear all completed</button>
+    </form>
```

Comment:
Kindly make sure you are following the design as the previous reviewer requested. Check this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) and use Webpack Style/CSS Loader to load your CSS.
<img width="217" alt="Screenshot 2023-03-04 at 3 41 39 PM" src="https://user-images.githubusercontent.com/25887253/222902082-1fba039f-5311-4ceb-b135-6ccbb4275c7a.png">